docs/UsersGuide/dev/testAPI/user_guide.md

```diff
@@ -719,7 +719,7 @@ I see two options to address this:
 
 ### The openpyxl library has thrown WorkbookAlreadySaved error
 
-While running unit tests on the API, there was an error thrown by openpyxl that caused the log to close early. The behavior wasn't able to be recreated, but the [Test Logger](../../../../Gds/src/fprime_gds/common/logger/test_logger.py) was updated to [catch the exception](https://github.jpl.nasa.gov/FPRIME/fprime-sw/blob/717bc6fab85c53680108fc961cad6338e779816f/Gds/src/fprime_gds/common/logger/test_logger.py#L124) to prevent tests from failing due to the logger.
+While running unit tests on the API, there was an error thrown by openpyxl that caused the log to close early. The behavior wasn't able to be recreated, but the [Test Logger](../../../../Gds/src/fprime_gds/common/logger/test_logger.py) was updated to [catch the exception](https://github.com/nasa/fprime/blob/717bc6fab85c53680108fc961cad6338e779816f/Gds/src/fprime_gds/common/logger/test_logger.py#L124) to prevent tests from failing due to the logger.
```

Comment:
There were a number of instances of what I presume are the NASA GHES. Since there appears to be a straightforward correspondence, it seemed worthwhile to suggest this change.

I found this because of a candidate pattern that I've chosen to disable -- the one for https/http/file urls.